presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -664,12 +664,7 @@ public static SchemaTableName schemaTableName(ConnectorTableHandle tableHandle)
 
     public static String createPartitionName(Partition partition, Table table)
```

Comment:
look at the current usages of this method and consider inlining it, since it is just a pass though to the new `MetastoreUtil`